blob/src/main/java/io/crate/blob/v2/BlobIndex.java

```diff
@@ -101,7 +109,7 @@ BlobShard removeShard(ShardId shardId) {
      * and return the root path of the blob directory.
      */
     private Path retrieveBlobRootDir(BlobShard blobShard) {
-        Path blobDir = blobShard.getBlobDir();
+        Path blobDir = blobShard.getBlobDataDir();
```

Comment:
you must not change this - it is used in other places too also the intent is to return the root dir not the data dir